projects/demo-playwright/tests/kit/input-month-range/input-month-range.spec.ts

```diff
@@ -0,0 +1,42 @@
+import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
+import {expect, Locator, test} from '@playwright/test';
+
+import {TuiInputMonthRangePO} from '../../../utils/page-objects/input-month-range.po';
```

Comment:
Add new PO to the following file:
https://github.com/taiga-family/taiga-ui/blob/main/projects/demo-playwright/utils/page-objects/index.ts

And move this import to `@demo-playwright/utils`.

Reply:
Changed this